### ISSUE TITLE
RSP not working 

### ISSUE THREAD

**Issue by singhalmradul** [redhat-developer/vscode-server-connector#585]
I'm using Linux Mint 21.1 Cinnamon and getting this prompt every time I try to create a server.
`Unable to start the server: Extension backend error - rsp error - community server connector failed to start - cannot read properties of null (reading 'rspreqreject')`
 I even tried installing older versions of the extension as I found in some solutions but that didn't work either. Thanks

**Comment by robstryker** [comment]
So... I can't replicate this issue. What's worse, I've looked over the code and can't see any way a null or undefined value makes it to the only two possible locations where a `rspreqreject` field is being read. 

That said, I'll be adding some null checks just to try to beef it up, but, this likely won't help discover the cause of your issue. If these null checks do get hit, it just means it's been an unexpected null check. Maybe the output can help discover what's wrong, but, maybe it can't.

**Comment by robstryker** [comment]
I've made a snap bugfix release to try to improve the error messaging here. I'll leave the issue open, though, since I'm almost 100% sure it won't fix your issue but only possibly reveal what's causing it.

**Comment by singhalmradul** [comment]
now it's the error for reading properties of undefined which is type 
`Unable to edit server properties Extension backend error - cannot read properties of undefined (reading 'type')`

**Comment by robstryker** [comment]
To be clear, you are getting an error message that reads `Unable to edit server properties` when you try to create a server? 

This error should only ever show up when trying to edit a server. Is this really the same issue or a different issue altogether?

**Comment by robstryker** [comment]
* 6b4c454 - (HEAD -> main, origin/main) Related to issue 585 - null checks (8 seconds ago) <Rob Stryker>


This will only address the null checks. There's gotta be something else going on though. 

Can you explain if the rsp works in most cases? Are these intermittent bugs? Do they happen randomly? Or only when doing specific actions? Does the RSP actually start? Are you able to create servers or start them?

**Comment by singhalmradul** [comment]
Yes, the error message that reads `Unable to edit server properties` appeared when I tried to create a server. RSP was working well on Ubuntu 22.04.2. I encountered this issue when I switched to Linux Mint 21.1 Cinnamon.

RSP wasn't working before but when I tried creating a server this time, I was able to create it. Now I'm not encountering the error when I'm creating servers. 

Thanks for fixing the issue.
